### PR TITLE
Parsers adjustments v1.2

### PR DESCRIPTION
**Pull request by Filari02**
fixed minor bugs in parsers. tested parsers many times and it seems they work fluently now. However i haven't tested Smartphone parsers yet

### CODE REVIEW

File: ExceptionsLogging/NotParsedElementsLog.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Could not parse processor. Processor ID is:   64
+Time:    11/26/2020 12:00:27 PM
+Could not parse processor. Processor ID is:   65
+Time:    11/26/2020 12:03:52 PM
+Could not parse processor. Processor ID is:   66
+Time:    11/26/2020 12:05:54 PM
+Could not parse processor. Processor ID is:   67
+Time:    11/26/2020 12:06:55 PM
+Could not parse processor. Processor ID is:   68
+Time:    11/26/2020 12:12:43 PM
```
Comment (Ugniusst) [inline ExceptionsLogging/NotParsedElementsLog.txt]: File could be deleted

File: DataContent/ReadingDB/ProcessorDataService.cs [hunk 2]
```diff
@@ -77,14 +77,18 @@ public Processor GetProcessor(string processorModel)
                         catch (ProcessorNotFoundException)
                         {
                             processor = new Processor { Model = processorModel };
-                            processor.SetName(processorModel);
-                            using (_db = new ComputerContext())
+                            if (processor.Model.Length < 31)
```
Comment (Ugniusst) [inline DataContent/ReadingDB/ProcessorDataService.cs]: should be < 33 (Max lenght is 32)

**Review by Tomas-Per — APPROVED** [review]
Approving now, but you'll have to run tests on the smartphone parsers ASAP

File: ExceptionsLogging/NotParsedElementsLog.txt (hunk 1 above)
Comment (Filari02) [reply ExceptionsLogging/NotParsedElementsLog.txt]: deleted

File: DataContent/ReadingDB/ProcessorDataService.cs (hunk 2 above)
Comment (Filari02) [reply DataContent/ReadingDB/ProcessorDataService.cs]: fixed

**Comment by Filari02** [comment]
i can't start testing, since we do not have smartphones DB

**Comment by Tomas-Per** [comment]
We do have a smartphones table in the database AND the API which should be used instead of the now obsolete services (I highly recommend looking into HttpClientFactory for this), since the changes to DB and the API implementation have not been merged yet I understand that the parsers couldn't be tested in this manner and as such we will have to merge this branch anyways.

**Pull request merged**